templates/playlist/playlist.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="container-fluid">
+    <div class="row">
+        <div class = "col-md-3">
+            <h2>{{playlist.title}}</h2>
```

Comment:
CONSISTENCY!

your templatetag format is `{%<space>variable<space>%}` but your other tags has no space. make up your mind